### DISCUSSION TITLE
官方文档打不开了？

### DISCUSSION THREAD

**Discussion by a145789** [ant-design/ant-design-mobile#5366] [General]
跳进去显示 500 Internal Server Error

**Comment by GOWxx** [comment]
现在可以访问了
https://mobile.ant.design/zh

**Comment by awmleer** [comment]
底层 CDN 服务 Cloudflare 出现了故障，导致了今天下午出现短暂无法访问的问题，现在看起来 Cloudflare 已经恢复了
